.travis.yml

```diff
@@ -33,10 +33,15 @@ script:
   - if [[ "$TRAVIS_GO_VERSION" == 1.12* && "$TRAVIS_OS_NAME" == "osx" ]]; then export GOFLAGS=-mod=vendor; fi
   - go test -tags ci ./...
   - go vet -tags ci -unsafeptr=false ./...
-  - golint -set_exit_status $(go list -tags ci ./...) || exit 1
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then exit; fi'
-  - go test -tags ci -covermode=atomic -coverprofile=coverage.out ./...
-  - $(go env GOPATH | awk 'BEGIN{FS=":"} {print $1}')/bin/goveralls -coverprofile=coverage.out -service=travis-ci -repotoken $COVERALLS_TOKEN
+  - golint -set_exit_status $(go list -tags ci ./...)
```

Comment:
Doesn't this remove the exit if lint fails?

Reply:
Travis respects the return codes of every command. That's why it is not necessary.

Reply:
My understanding was that it records the output result but progresses - and so the following code would be run?

Reply:
Your understanding is correct. Why should the coverage part should be skipped if the linter complains?